.github/workflows/main.yml

```diff
@@ -20,7 +20,15 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         args: --all-features -- -D warnings
-  test:
+  test_ledger:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: docker pull docker.io/wszdexdrf/ledger_emulator
```

Comment:
Same as the comment above, don't pull from docker.io, use the Dockerfile you already have